packages/integrations/package.json

```diff
@@ -28,7 +28,8 @@
     "@configu/sdk": "workspace:*",
     "lodash-es": "^4.17.21",
     "reflect-metadata": "^0.1.13",
-    "typeorm": "^0.3.17"
+    "typeorm": "^0.3.17",
+    "@openfeature/server-sdk": "^1.7.4"
```

Comment:
This shouldn't be a dependency within the root of the integrations `package.json`. It's enough that it's included in the `package.json` of the specific integration.